util/pkg/distributions/distributions.go

```diff
@@ -99,6 +99,13 @@ func (d *Distribution) HasLoopbackEtcResolvConf() bool {
 		// Ubuntu > 16.04 has it
 		return d.version > 16.04
 	}
+	if d.project == "flatcar" {
+		// Before version 2748.0.0. the "/etc/resolv.conf" file did not point to the stub resolver
+		// configuration. Instead, it shared content (by way of a symbolic link) with systemd's
+		// "/run/systemd/resolve/resolv.conf" file, as opposed to the
+		// "/run/systemd/resolve/stub-resolv.conf" file. The latter includes the loopback address.
+		return d.version >= 2748.0
+	}
```

Comment:
kOps does not set the flatcar version it finds, so comparing with `2748.0` will not help much.
Being a rolling release and `2748.0` is no longer listed in https://www.flatcar.org/releases, I think it's safe to consider this true for Flatcar in general.
Also, not sure if the long comment is needed considering the function comment.
```suggestion
	if d.IsFlatcar() == "flatcar" {
		// Before version 2748.0.0. the "/etc/resolv.conf" file did not point to the stub resolver
		// configuration. Instead, it shared content (by way of a symbolic link) with systemd's
		// "/run/systemd/resolve/resolv.conf" file, as opposed to the
		// "/run/systemd/resolve/stub-resolv.conf" file. The latter includes the loopback address.
		return true
	}
```

Reply:
I had originally written it as you suggest, then stopped to go figure out why it hadn't been necessary to have this change in place in much older versions of Flatcar. Thinking about it again, though, while it wasn't _necessary_ to use the alternate _resolv.conf_ file in those older versions, it would have worked to do so. In other words, using either of the candidate files worked equally well, as far as I can tell.